huntsman/utils/light_control.py

```diff
@@ -0,0 +1,265 @@
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username
+
+#note, light 5 can be 'uncommented' once installed, or causes problems using currently 
+
+
+#deal with error system - should errors be sent to pocs? how does pocs handle error messages?
+
+def manual_login_phillips_hue(hue_ip="10.88.21.10", verbose = False):
+    
+    """
```

Comment:
Same pep8 and docstring comments as in #29

Reply:
I used Autopep8 in spyder and it seemed to work

Reply:
> I used Autopep8 in spyder and it seemed to work

You did this before and this is how the code looks, or you have since done this? If you had already done then something is not correct about it as there are many pep8 errors.

Reply:
I've since done this, so should be corrected now for the next pr